interfaces/policy/basedeclaration.go

```diff
@@ -146,535 +146,10 @@ revision: 0
-        - core
-    deny-auto-connection: true
-  process-control:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  pulseaudio:
-    allow-installation:
-      slot-snap-type:
-        - app
-        - core
-    deny-connection:
-      on-classic: false
-  raw-usb:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  removable-media:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  screen-inhibit-control:
-    allow-installation:
-      slot-snap-type:
-        - core
-  serial-port:
-    allow-installation:
-      slot-snap-type:
-        - core
-        - gadget
-    deny-auto-connection: true
-  shutdown:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  snapd-control:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  storage-framework-service:
-    allow-installation:
-      slot-snap-type:
-        - app
-    deny-connection: true
-    deny-auto-connection: true
-  system-observe:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  system-trace:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  thumbnailer-service:
-    allow-installation:
-      slot-snap-type:
-        - app
-    deny-auto-connection: true
-    deny-connection: true
-  time-control:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  timeserver-control:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  timezone-control:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  tpm:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  udisks2:
-    allow-installation:
-      slot-snap-type:
-        - app
-    deny-connection: true
-    deny-auto-connection: true
-  uhid:
-    allow-installation:
-      slot-snap-type:
-        - core
-    deny-auto-connection: true
-  unity7:
-    allow-installation:
-      slot-snap-type:
-        - core
-  unity8:
-    allow-installation:
-      slot-snap-type:
-        - app
-    deny-connection: true
-  unity8-calendar:
-    allow-installation:
-      slot-snap-type:
-        - app
-    deny-auto-connection: true
-    deny-connection: true
-  unity8-contacts:
-    allow-installation:
-      slot-snap-type:
-        - app
-    deny-auto-connection: true
-    deny-connection: true
-  ubuntu-download-manager:
-    allow-installation:
-      slot-snap-type:
-        - app
-    deny-connection: true
-  upower-observe:
-    allow-installation:
-      slot-snap-type:
-        - core
-        - app
-    deny-connection:
-      on-classic: false
-  x11:
-    allow-installation:
-      slot-snap-type:
-        - core
```

Comment:
I wasn't sure what is the real purpose of all these changes until I reached this... Very nice, it's great we get rid of this big blob :)